src/ast/passes/semantic_analyser.cpp

```diff
@@ -1361,7 +1365,9 @@ void SemanticAnalyser::visit(Call &call)
       }
     } else if (arg.type.IsIntTy() && arg.is_literal) {
       auto sig = bpftrace_.get_int_literal(&arg);
-      if (!sig.has_value() || *sig < 1 || *sig > 64) {
+      if (!sig.has_value())
+        LOG(ERROR, call.loc, err_) << call.func << ": invalid signal value";
```

Comment:
I had to go with `LOG(ERROR)` here b/c while the above `get_int_literal` does `LOG(ERROR)`, it doesn't print to `err_` so semantic analyser doesn't properly fail.